public/scss/components/_buttons.scss

```diff
@@ -71,22 +71,36 @@ a[role='button'] {
 }
 
 .back-button {
-  margin-bottom: $space-md;
-  padding: 0;
-  text-decoration: none;
   display: inline-block;
-  border-bottom: 1px solid;
-  font-size: 0.9em;
+  position: relative;
+  margin-top: 15px;
+  margin-bottom: 15px;
+  padding-left: 16px;
```

Comment:
Can we use our spacing units where they make sense? I think `$space-md` is 15 pixels.